jax/api.py

```diff
@@ -1198,12 +1197,12 @@ def vmap(fun: F, in_axes=0, out_axes=0, axis_name=None) -> F:
   in_axes_, out_axes_ = tree_leaves(in_axes), tree_leaves(out_axes)
   if not all(isinstance(l, (type(None), int)) for l in in_axes_):
     msg = ("vmap in_axes must be an int, None, or (nested) container with "
-           "those types as leaves, but got {}.")
-    raise TypeError(msg.format(in_axes))
+           f"those types as leaves, but got {in_axes}.")
+    raise TypeError(msg)
   if not all(isinstance(l, (type(None), int)) for l in out_axes_):
     msg = ("vmap out_axes must be an int, None, or (nested) container with "
-           "those types as leaves, but got {}.")
-    raise TypeError(msg.format(out_axes))
+           f"those types as leaves, but got {out_axes}.")
+    raise TypeError(msg)
```

Comment:
nit: remove the `msg` variable here, as you did elsewhere?

Reply:
Done.